packetery/src/Packetery/Checkout.php

```diff
@@ -268,17 +264,28 @@ public function validatePickupPointData(): void {
 	/**
 	 * Saves pickup point information to order
 	 *
-	 * @param int $order_id Order id.
+	 * @param int $orderId Order id.
 	 */
-	public function updateOrderMeta( int $order_id ): void {
+	public function updateOrderMeta( int $orderId ): void {
+		$chosenMethod = $this->getChosenMethod();
+		if ( false === $this->isPacketeryOrder( $chosenMethod ) ) {
+			return;
+		}
+		// TODO: use Entity::META_CARRIER_ID.
```

Comment:
Proč tuto konstantu nepoužijeme/nezavedeme hned ? Co nám v tom brání, že musíme vytvářet další TODO ?

Reply:
Protože už je to připraveno v https://github.com/Zasilkovna/WooCommerce/pull/38